.travis.yml

```diff
@@ -3,66 +3,47 @@
 # language: python causes grief on osx
 language: generic
 os:
-  # - osx
-   - linux
+   - osx
+  # - linux
 env:
   - TRAVIS_PYTHON_VERSION=2.7 MPIRUN= FIPY_SOLVERS=pysparse
-  # - TRAVIS_PYTHON_VERSION=2.7 MPIRUN= FIPY_SOLVERS=pysparse FIPY_INLINE=1
+#  - TRAVIS_PYTHON_VERSION=2.7 MPIRUN= FIPY_SOLVERS=pysparse FIPY_INLINE=1
   - TRAVIS_PYTHON_VERSION=2.7 MPIRUN= FIPY_SOLVERS=scipy
   - TRAVIS_PYTHON_VERSION=2.7 MPIRUN= FIPY_SOLVERS=trilinos
   - TRAVIS_PYTHON_VERSION=2.7 MPIRUN="mpirun -np 2" FIPY_SOLVERS=trilinos
   - TRAVIS_PYTHON_VERSION=3.6 MPIRUN= FIPY_SOLVERS=scipy
 cache: apt
-addons:
-  apt:
-    packages:
-    # - liblapack3
-    # - liblapack.so.3
-    # - libatlas-dev
-    # - libatlas-base-dev
-    # - liblapack-dev
-    # - liblapack3gf
-    # - gfortran
 branches:
   except:
     - nist-pages
 before_install:
   # sudo ln -s /usr/lib/lapack/liblapack.so.3gf /usr/lib/lapack/liblapack.so.3;
   # sudo ldconfig;
-  - if [[ $TRAVIS_OS_NAME -eq "linux" ]]; then
+  - if [[ $TRAVIS_OS_NAME = "linux" ]]; then
```

Comment:
Surprised that `=` is [technically correct](https://www.gnu.org/software/bash/manual/html_node/Bash-Conditional-Expressions.html). Makes my brain twitch, but OK!

Reply:
I don't know if it is, but `-eq` is very much not correct